From 7f8ef553b7c433af153d48c6a16b2943780abf67 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sat, 31 Dec 2022 14:41:52 -0800
Subject: [PATCH] install: Do not undef _FILE_OFFSET_BITS

_FILE_OFFSET_BITS is a feature test macro to determine largefile
support. Usually its set to 64 on systems supporting LFS. Its also
needed to be set to 64 for supporting 64bit time_t on glibc on 32bit
systems. If its undefined explicitly, then 64bit time_t can not be
enabled.

Upstream-Status: Submitted [https://github.com/dracutdevs/dracut/pull/2157]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/install/dracut-install.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/install/dracut-install.c b/src/install/dracut-install.c
index dda0caca..05a67a03 100644
--- a/src/install/dracut-install.c
+++ b/src/install/dracut-install.c
@@ -22,7 +22,6 @@
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
-#undef _FILE_OFFSET_BITS
 #include <ctype.h>
 #include <errno.h>
 #include <fcntl.h>
-- 
2.39.0

